### DISCUSSION TITLE
Beta psp emulator

### DISCUSSION THREAD

**Discussion by ethanaobrien** [EmulatorJS/EmulatorJS#569] [General]
New ***highly beta*** psp core pushed.
As stated in the EmulatorJS readme:
> PSP is not yet supported. Some of y'all may have seen that I pushed a "beta" ppsspp core, but this core is not ready for daily use. It still crashes randomly and any games that use 3d (so like, all of them) will just have a white screen (and might just crash). Do not open issues related to the "psp" core.

This core can be tested without setting up your own instance at the site https://testing.emulatorjs.org. if you add the psp core to your site, it is recommended to fully test the game first, and note it doesn't work on most mobile devices because of the amount of required ram.

It seems my initial conclusion of 3d was incorrect. Some 3d games seem to work

**Comment by daxinhub** [comment]
hello,Where can I find the wasm.data for PSP
